README.md

```diff
@@ -56,12 +56,10 @@ ______________________________________________________________________
 
 ## Installation
 
-The current recommended way to install Refiners is from source using [Rye](https://rye-up.com/):
+The current recommended way to install Refiners is from source:
 
 ```bash
-git clone "git@github.com:finegrain-ai/refiners.git"
```

Comment:
I would keep this. And just add:

Alternatively:

```bash
pip install git+https://github.com/finegrain-ai/refiners.git
```

Reply:
@Laurent2916 wdyt?

Reply:
I feel like the whole "rye setup" is more "dev oriented" than "user oriented".
wdyt about simply adding a reference to the CONTRIBUTING.md if people want to setup a dev env ?

Reply:
True. But https://refine.rs/getting-started/recommended/ is rye-centric.

Reply:
Anyway, let's rebase and use one or the other way.

Reply:
The recommended option on the website is Rye because you needed to convert the weights anyway. We could revisit this.

Reply:
Ah right, there is still the weight conversion. I agree we should probably revisit this before modifying the README.